lib/tackle/connection.ex

```diff
@@ -115,5 +115,22 @@ defmodule Tackle.Connection do
     {:error, :no_process}
   end
 
-  def open(url), do: AMQP.Connection.open(url)
+  @spec open(String.t()) :: {:ok, AMQP.Connection.t()} | {:error, atom()} | {:error, any()}
+  def open(url) do
+    AMQP.Connection.open(url)
+  end
+
+  # Saving OS certs during compile time. Fetching them during runtime results in
+  # a {:EXIT, #Port<0.44>, :normal} message received by the Executor GenServer.
+  # @certs :public_key.cacerts_get()
```

Comment:
I think the proper way to handle this would be to handle the port message. Or raise this issues with OTP (https://github.com/erlang/otp/blob/0f4345094b9a57c4166c695b7acbb6087df7e444/lib/public_key/src/pubkey_os_cacerts.erl#L133)